src/main/play-26/twirl/uk/gov/hmrc/govukfrontend/views/components/govukInput.scala.html

```diff
@@ -65,7 +65,7 @@
 
 @inputSnippet(describedBy: String) = {
   @defining {
-    id="@id" name="@name" type="@inputType"@value.mapNonEmpty { value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@describedBy"}
+    id="@id" name="@name" type="@inputType"@if(spellcheck.nonEmpty){ spellcheck="@if(spellcheck.getOrElse(true)){true}else{false}" }@value.mapNonEmpty { value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@describedBy"}
```

Comment:
This can be simplified to:
@if(spellcheck.nonEmpty) { spellcheck="@spellcheck"}

Twirl templates will automatically unwrap the value inside an Option:
https://www.playframework.com/documentation/2.7.x/ScalaTemplates#Displaying-Scala-types

Reply:
Woohoo, I'd hoped that was possible.